src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-01-01__create_itm_contrib_opt_conf_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE item_contribution_options_configuration
+(
+    central_server_id UUID       NOT NULL PRIMARY KEY,
```

Comment:
`NOT NULL` is not required for primary key, it's "not null" by default

Reply:
On the second though, to keep consistent with other existing tables please leave it as is